codec-http/src/main/java/io/netty/handler/codec/http/multipart/AbstractDiskHttpData.java

```diff
@@ -114,21 +114,18 @@ public void setContent(ByteBuf buffer) throws IOException {
             }
             if (buffer.readableBytes() == 0) {
                 // empty file
-                if (!file.createNewFile()) {
-                    throw new IOException("file exists already: " + file);
```

Comment:
Can you explain why you think this should not happen ?

Reply:
hmm, because this line that is before

```java
if (file == null) {
      file = tempFile();
}
```

I am trying to find such case when this removed line would be necessary, but actually I cannot :) If you know such case (I am newbie here, of course I can be wrong) I would be happy to provide test case for that.

P.S 
how I understand this (this is about version before my changes)
1. someone will pass empty buffer as parameter.
2. temp file is created
3. check if buffer is empty 
4. buffer is empty, create new file -> results in IOException

I don't get it :/

Reply:
@normanmaurer ping :)